### ISSUE TITLE
Add back the server browser button?

### ISSUE THREAD

**Issue by credomane** [raysfire/rayshud#110]
I quit playing TF2 for a year or so. Came back updated Rayshud only to discover the server browser button was gone. Browsing around the commits and it seems that the it was removed with the ScreemFortress 2017 update.
Any reason this was done?
Can we add it back?

**Comment by credomane** [comment]
I did a quick edit on my local copy to get it back. Patch pasted here.

    diff --git a/resource/GameMenu.res b/resource/GameMenu.res
    index 1377990..e771269 100644
    --- a/resource/GameMenu.res
    +++ b/resource/GameMenu.res
    @@ -47,6 +47,13 @@
            "subimage"		"glyph_options"
            "tooltip"		"Console"
        }
    +	"ServerBrowserButton"
    +	{
    +		"label"			""
    +		"command"		"openserverbrowser"
    +		"subimage"		"glyph_options"
    +		"tooltip"		"Server Browser"
    +	}
        "CallVoteButton"
        {
            "label"			""
    diff --git a/resource/ui/MainMenuOverride.res b/resource/ui/MainMenuOverride.res
    index d3ec8c2..0bb9ec6 100644
    --- a/resource/ui/MainMenuOverride.res
    +++ b/resource/ui/MainMenuOverride.res
    @@ -1108,6 +1108,76 @@
                }
            }
        }
    +
    +	"ServerBrowserButton"
    +	{
    +		"ControlName"	"EditablePanel"
    +		"fieldname"		"ServerBrowserButton"
    +		"xpos"			"205"
    +		"ypos"			"62"
    +		"zpos"			"1"
    +		"wide"			"20"
    +		"tall"			"20"
    +		"autoResize"	"0"
    +		"pinCorner"		"3"
    +		"visible"		"1"
    +		"enabled"		"1"
    +		"tabPosition"	"0"
    +
    +		"SubButton"
    +		{
    +			"ControlName"	"CExImageButton"
    +			"fieldName"		"SubButton"
    +			"xpos"			"0"
    +			"ypos"			"0"
    +			"wide"			"20"
    +			"tall"			"20"
    +			"zpos"			"5"
    +			"autoResize"	"0"
    +			"pinCorner"		"3"
    +			"visible"		"1"
    +			"enabled"		"1"
    +			"tabPosition"	"0"
    +			"labelText"		""
    +			"font"			""
    +			"Command"		"openserverbrowser"
    +			"textAlignment"	"center"
    +			"dulltext"		"0"
    +			"brighttext"	"0"
    +			"default"		"1"
    +
    +			"sound_depressed"	"UI/buttonclick.wav"
    +			"sound_released"	"UI/buttonclickrelease.wav"
    +
    +			"actionsignallevel" "2"
    +			"proportionaltoparent"	"1"
    +
    +			"paintbackground"	"0"
    +			"paintborder"		"0"
    +
    +			"defaultFgColor_override" "TanLight"
    +			"armedFgColor_override" "Button Color"
    +			"depressedFgColor_override" "Button Color"
    +
    +			"image_drawcolor"	"235 226 202 255"
    +			"image_armedcolor"	"72 255 255 255"
    +
    +			"SubImage"
    +			{
    +				"ControlName"	"ImagePanel"
    +				"fieldName"		"SubImage"
    +				"xpos"			"3"
    +				"ypos"			"1"
    +				"zpos"			"1"
    +				"wide"			"15"
    +				"tall"			"15"
    +				"visible"		"1"
    +				"enabled"		"1"
    +				"scaleImage"	"1"
    +				"image"			"glyph_create"
    +			}
    +		}
    +	}
        
        "ReportPlayerButton"
        {

**Comment by CriticalFlaw** [comment]
It was removed because it was accessible through the matchmaking side panel when you click Find Game.

Having the button that does the same thing in two places is redundant and would only prompt people to ask to add other button like Casual, Competitive MVM, and Create a Server as well which would only bloat the main menu

**Comment by credomane** [comment]
Wow...I don't think I ever would have discovered that myself. Even though I read every option there I was convinced it was all purely "Matchmaking" crap. So I just went straight to pasting `gamemenucommand openserverbrowser` into the console to solve my problem.

I finally got tired of that and made this bug report while digging through the code to re-add the button. No need to add a button that is actually there. Closing this issue and going to hide in a corner.
